common/lib/xblock/xblock_discussion/xblock_discussion.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+"""
+Discussion XBlock
+"""
+import json
+import logging
+
+from xblockutils.resources import ResourceLoader
+from xblockutils.studio_editable import StudioEditableXBlockMixin
+
+from xblock.core import XBlock
+from xblock.fields import Scope, String, UNIQUE_ID
+from xblock.fragment import Fragment
+
+log = logging.getLogger(__name__)
+LOADER = ResourceLoader(__name__)
```

Comment:
I decided to just suppress the warning.  `loader` isn't used anywhere else, so it seems overkill to push the change into `pylintrc_tweaks`.

Fixed in [53a515e](https://github.com/edx/edx-platform/pull/12582/commits/53a515e0c59b6d67c0f4bcfcb2cfcd35bade2e10).